modules/proxy-namespace-procedure.adoc

```diff
@@ -0,0 +1,42 @@
+
+[[red-hat-quay-proxy-namespace-procedure]]
+= Using {productname} to proxy a namespace in a registry
+
+The following procedure describes how you can use {productname} to proxy a namespace in a registry. This procedure is set up to proxy quay.io, which allows users to use `podman` to pull any public image from any namespace on quay.io.
+
+.Prerequisites
+
+*
+
+*
+
+.Procedure
+
+. In your Quay organization on the UI, for example, `cache-quayio`, click *Organization Settings* on the left hand pane.
+
+. In *Remote Registry* enter the name of the remote registry to be cached, for example, `quay.io`, and click *Save*.
++
+[NOTE]
+====
+* By adding a namespace to the *Remote Registry*, for example, `quay.io/<namespace>`, users in your organization will only be able to proxy from that namespace.
+* If you do not set a *Remote Registry Username* and *Remote Registry Password*, you cannot add one without removing the proxy cache and creating a new registry.
+* The default *Expiration* for cached images is set to 86400 seconds. This value can be adjusted. but is refreshed every pull. Un-refreshed images will disappear after the allotted time, but are still stored in Quay. The time in which an image is completely deleted, or garbage collected, depends on the *Time Machine* setting of your organization. The default time for garbage collection is 14 days unless otherwise specified.
```

Comment:
Can you elaborate on the sentence below?
> This value can be adjusted. but is refreshed every pull.

The "but" here has me thinking that even though the user may set an expiration other than the default 86400, Quay will overwrite it with the default 86400 on every pull.

In case there's confusion on how this works (because I think this is confusing): "expiration" is the default _tag_ expiration Quay sets for all cached images. The tag expiration is renewed (reset, or refreshed) every time the tag is pulled. It's possible to override individual tag expiration (this is a Quay feature, independent of proxy), but since Quay renews this value based on the proxy org's "expiration" config, individual tag expiration overrides will only last until the tag's next pull, when Quay will reset it to whatever the proxy org's "expiration" config is set to.

Also: you totally nailed the interaction of the cache with the time machine config! 🚀